oaknational/Oak-Web-Application
# style: card link with icon focus styles

## Description

- Adds a transitions folder with variable for `HOVER_SHADOW_TRANSITION` so that all hover shadow transitions should be in sync
- Adds focus styles to the `CardLinkIcon` component

## Issue(s)

Progresses #434 

## How to test

1. Go to {cloud link}
2. Focus on the "Develop your curriculum" link
3. You should see :

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/189121959-1b08b1e9-9137-4af4-9673-6954b2418f3e.png)


## Checklist
